portfolio/src/main/webapp/script.js

```diff
@@ -1,34 +1,299 @@
 // Author: Jonathan Chavez - tabaresj@google.com
 var isTerminalOpen = false;
+var firstTimeOpening = true;
+var currentSection = 'whoami';
+
+// Names for the sites in the terminal, each is a section of the portfolio
+var sites = [
+  'education.yaml',
+  'work-experience.yaml',
+  'projects.yaml',
+  'skills.yaml',
+  'virus.exe',
+];
+
+// List of commands available to use in the terminal
+var commands = ['man', 'compgen', 'ls', 'cat', 'whoami'];
 
 window.addEventListener('keydown', keyboardHandler);
 
+/*
+ * Handles keyboard strokes for detecting when toggling the terminal
+ */
 function keyboardHandler(e) {
   e = e || window.event;
-  if (String.fromCharCode(e.keyCode) === 'T') {
+  if (String.fromCharCode(e.keyCode) === 'T' && e.shiftKey) {
     if (!isTerminalOpen) {
       openTerminal();
       isTerminalOpen = true;
-      runTerminal();
+      if (firstTimeOpening) {
+        firstTimeOpening = false;
+        runTerminalTutorial();
+      }
     } else {
       closeTerminal();
       isTerminalOpen = false;
     }
   }
 }
 
-function runTerminal() {}
+/*
+ * Shows a tutorial on the terminal
+ */
+async function runTerminalTutorial() {
+  await printTerminal("^500Hola! ^400 I'm Jonathan,^300 nice to meet you.");
```

Comment:
According to https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes, strings should use single quotes. But maybe in this case you want to use string templates instead to avoid escaping the single quotes you use in the strings below.

Reply:
Done. Thanks for letting me know that.